mica-core/src/main/java/org/obiba/mica/config/UpgradeConfiguration.java

```diff
@@ -51,6 +52,7 @@ public UpgradeManager upgradeManager(MicaVersionModifier micaVersionModifier,
   public List<UpgradeStep> upgradeSteps(ApplicationContext applicationContext) {
     return Lists.newArrayList(applicationContext.getBean(AttachmentsRefactorUpgrade.class), //
       applicationContext.getBean(NetworkLogoMigration.class), //
+      applicationContext.getBean(NetworkStateUpgrade.class), //
```

Comment:
Do you have an idea why this was removed by Alex?
https://github.com/kazoompa/mica2/commit/209ed34e0e0b390514dbd9f0342ee26bbc4d09da#diff-750d2de22fc92ff6bef4f10c46940c82